### ISSUE TITLE
mycinema.pro: breakage

### ISSUE THREAD

**Issue by Anuj009** [uBlockOrigin/uAssets#13001]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625) (also mind browser built-in blockers)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://www.mycinema.pro/`

### Category

breakage

### Describe the issue

uBlock blocks some scripts on the webpage which I'm unable to unblock manually. These scripts relate to the actual tool functionality for which "Magic Actions for YouTube" is used, like in this case, the second element that is blocked by uBlock is the screenshot editor or YouTube so I'm unable to access the screenshots taken.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![Screenshot-Composer-Magic-Software](https://user-images.githubusercontent.com/15170189/165723188-f4a7f247-f33c-4f38-a9e8-312d32a8e517.png)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.42.4
Chromium: 100
filterset (summary): 
  network: 89058
  cosmetic: 0
  scriptlet: 18177
  html: 0
listset (total-discarded, last updated): 
  added: 
    https://secure.fanboy.co.nz/fanboy-annoyance_ubo.txt: 78705-69948, 4m
  default: 
    user-filters: 0-0, never
    easylist: 66621-34422, 6m
    easyprivacy: 26909-218, 6m
    plowe-0: 3679-805, 4m
    ublock-abuse: 75-1, 6m
    ublock-badware: 4109-492, 6m
    ublock-filters: 30976-7843, 6m
    ublock-privacy: 213-13, 6m
    ublock-quick-fixes: 183-73, 6m
    ublock-unbreak: 1767-267, 6m
    urlhaus-1: 8236-0, 6m
filterset (user): [empty]
trustedset: 
  added: [array of 4 redacted]
modifiedUserSettings: 
  parseAllABPHideFilters: false
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 640 ms (selfie)
  maxAssetCacheWait: 881 ms
popupPanel: 
  blocked: 5
  blockedDetails: 
    api64.com: 1
    google.co.in: 1
    google.com: 1
    googlesyndication.com: 1
    googletagmanager.com: 1
```
</details>

**Comment by okiehsch** [comment]
I am not sure I understand the issue.
Your screenshot shows an error message but if I disable uBO and reload the sitein place of the error message I get ads
![image](https://user-images.githubusercontent.com/16838044/165724638-7fa509eb-0acd-49fd-b455-74746a8f6625.png)

What functionality of the site is broken and how exactly can I reproduce.
If I need to install an extension to be able to reprodcue some kind of breakge, this issue is out of scope and won't be fixed.

**Comment by Anuj009** [comment]
As you rightly pointed out, this issue needs the "Magic actions for YouTube" extension installed. As this makes the issue out of scope can you tell me how I can manually enable some script that is disabled by uBlock so I can try to fix it myself, please?

**Comment by okiehsch** [comment]
It really shouldn't interfere,  atleast globally, does it ony happen on certain websites, or does the extension not work on all websites?

**Comment by stephenhawk8054** [comment]
> As you rightly pointed out, this issue needs the "Magic actions for YouTube" extension installed. As this makes the issue out of scope can you tell me how I can manually enable some script that is disabled by uBlock so I can try to fix it myself, please?

Thing is you might need to describe the steps to trigger the issues in more details so others can reproduce and help you.

**Comment by okiehsch** [comment]
Reading about the extension it's a custom youtube video player that is ad supported.
Maybe blocking those ads is causing the issue.
We would need a screenshot of your logger after you reprouced the issue to be able to help.

**Comment by Anuj009** [comment]
The extension is not a video player but like a plugin for YouTube itself. The steps to reproduce the issue are as follows:

1. Download the chrome extension from here:
[https://chrome.google.com/webstore/detail/magic-actions-for-youtube/abjcfabbhafbcdfjoecdgepllmpfceif?hl=en](url)

2. Open this link on chrome and play the video:
[https://www.youtube.com/watch?v=XC5PPazhpg4](url)

3. You'll notice an icon with a magic wand in red under the seek bar of the video player. Click this and then the camera icon in the sub-menu to take a snapshot. This will take you to a different site on a new tab.
Ideally, this tab should show options for editing the snapshot that you just took on youtube but now I only get 2 blocks saying that the content was blocked by an adblocker.

Adding the logger markdown below
`<details><summary>Logger output</summary>

```
+6
www.mycinema.pro
3
ping
https://analytics.google.com/g/collect?v=2&tid=G-DJV2L5ZE22&gtm=2oe4p0&_p=1299767938&_z=ccd.NbB&gcs=G111&cid=1612865078.1651137133&ul=en-gb&sr=1366x768&dl=https%3A%2F%2Fwww.mycinema.pro%2Fcaptured-video-frames.html&uid=e69c97c0-58ff-4fd5-8934-7d6e2a6c58f7&sid=1651140703&sct=2&seg=1&dt=Screenshot%20Composer%20%7C%20Magic%20Software&_s=1
```
```
+3
www.mycinema.pro
1
image
https://www.mycinema.pro/i/e449w.png
```
```
+1
www.mycinema.pro
1
image
https://www.mycinema.pro/favicon.ico
```
```
+1
google-analytics.com/analytics.js
<<
www.mycinema.pro
3
script
https://www.google-analytics.com/analytics.js
```
```
+1
||google-analytics.com/analytics.js$script,redirect-rule=google-analytics.com/analytics.js
--
www.mycinema.pro
3
script
https://www.google-analytics.com/analytics.js
```
```
+1
||google-analytics.com/analytics.js$script
--
www.mycinema.pro
3
script
https://www.google-analytics.com/analytics.js
```
```
+1
www.mycinema.pro
3
script
https://www.googletagmanager.com/gtag/js?id=UA-326771-9&l=dataLayer&cx=c
```
```
+1
www.mycinema.pro
3
script
https://www.googletagmanager.com/gtag/js?id=G-DJV2L5ZE22
```
```
+1
www.mycinema.pro
1
image
https://www.mycinema.pro/i/get-firefox-extension.png
```
```
+1
www.mycinema.pro
1
image
https://www.mycinema.pro/i/get-chrome-extension.png
```
```
+1
www.mycinema.pro
1
image
https://www.mycinema.pro/i/promo-pic.png
```
```
+0
www.mycinema.pro
3
script
https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=ca-pub-0740458829455055
```
```
+0
www.mycinema.pro
1
image
https://www.mycinema.pro/i/magic-logo.png
```
```
+0
@@||mycinema.pro^$generichide
++
www.mycinema.pro
1
generichide
https://www.mycinema.pro/captured-video-frames.html
```
```
+0
@@||mycinema.pro^$generichide
++
www.mycinema.pro
1
generichide
https://www.mycinema.pro/captured-video-frames.html
```
```
+0
www.mycinema.pro
1
doc
https://www.mycinema.pro/captured-video-frames.html
```
```
+0
https://www.mycinema.pro/captured-video-frames.html
```
</details>
`

**Comment by stephenhawk8054** [comment]
@Anuj009 Does this filter works for you?

`@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,domain=mycinema.pro`

**Comment by Anuj009** [comment]
@stephenhawk8054 that filter doesn't work for me but I want to ensure that I added this filter correctly in the first place. Can you please let me know if I followed the right steps, please?

Open uBlock settings > "My filters" tab > _paste the above filter_ > Apply changes > Reload the broken website

**Also, I've taken a screenshot of the console log if it helps.**

![image](https://user-images.githubusercontent.com/15170189/165733986-7bba9152-a8a4-4208-83c4-9d1cc04e48eb.png)

**Here's a screenshot of the script in the Sources panel:**

![image](https://user-images.githubusercontent.com/15170189/165734533-b041d373-80ce-4e24-8929-52fad1fc23a1.png)

**Here's the script:**

`// Browser Extension - Magic Actions for YouTube™ - MYCINEMA.PRO - Copyright 2021 Vlad and Serge Strukoff - All Rights Reserved.
// Minified by Google Closure Compiler 20210406
'use strict';(function(){var v;const m=window,r=m.document;var e=Function.prototype.call;const t=function(){},F=CustomEvent,G=XMLHttpRequest,{runtime:{sendMessage:w,onMessage:x}}=chrome,H=x.addListener.bind(x),I=r.createElement.bind(r),y=e.bind(EventTarget.prototype.addEventListener),J=e.bind(EventTarget.prototype.dispatchEvent),K=e.bind(XMLHttpRequest.prototype.open),L=e.bind(XMLHttpRequest.prototype.send),q=e.bind(Array.prototype.push),M=e.bind(Array.prototype.join),z=e.bind(String.prototype.charCodeAt),
u=JSON.stringify,N=JSON.parse,A=String.fromCharCode,B=Date.now,C=m.postMessage.bind(m),D=function(a,c,n){var b=a?a:{w:640,h:350,vid:"pwHX9JT2APQ",lic:"YouTube License",t:"Magic Actions for YouTube™"};b.ver="7.9.5.1";b.ua=1;b.id=2001;"mozInnerScreenX"in m?C(b,"https://www.mycinema.pro"):(a=new G,K(a,"GET",b.rgb,!0),a.responseType="arraybuffer",y(a,"load",function(f){f instanceof Event&&f.isTrusted&&200==this.status&&(b.rgb=this.response,C(b,"https://www.mycinema.pro"))},!1),L(a))};e=function(a){if(a&&
a.detail&&a.detail.length){try{var c=a.detail;a=[];for(var n=[3,0,0,-3],b,f=0,p=c.length;f<p;f++)b=n[f%4],f+b<p?q(a,A(c[f+b])):q(a,A(c[f]));var d=M(a,"");var g=N(d)}catch(O){g={}}if(g.t&&1E3>B()-g.t)if(d=g.i,c=g.d,2===d)w(u({id:2,u:c.u,a:c.a,i:c.i}),t);else if(11===d){if(g.r){d={i:1,r:g.r,d:"7.9.5.1"};d.t=B();g=E;d=u(d);c=[];b=[3,0,0,-3];a=0;for(n=d.length;a<n;a++)p=b[a%4],a+p<n?q(c,z(d,a+p)):q(c,z(d,a));J(g,new F("lw",{detail:c}))}}else 2E3!==d||v||(v=!0,H(D),w(u({id:9}),D))}};if(m){for(var h in chrome)if("runtime"===
h)for(var l in chrome.runtime)"function"===typeof chrome.runtime[l]&&"connect"!==l&&(chrome.runtime[l]=t);else if("object"===typeof chrome[h])try{Object.defineProperty(chrome,h,{configurable:!1,enumerable:!1,writable:!1,value:{}})}catch(a){}var k="object"==typeof browser?browser:null;if(k)for(h in k)if("runtime"==h)for(l in k.runtime)"function"===typeof k.runtime[l]&&(k.runtime[l]=t);else if("object"===typeof k[h])try{Object.defineProperty(k,h,{configurable:!1,enumerable:!1,writable:!1,value:{}})}catch(a){}var E=
"noModule"in HTMLScriptElement.prototype?I("m-r"):r;y(E,"lc",e,!0)}})();
`

**Comment by stephenhawk8054** [comment]
@Anuj009 I can see the page normally after adding that filter

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/66517106/165736665-ac9d81d4-95ec-4a8c-b7a3-5cbf7464b0cb.png)

</details>

Chromium sometimes has some issues with loading the filters. You can try closing the browser first, then open again. Open the logger first before loading youtube and clicking on the extension button to ensure that that filter is loaded properly.

The log should looks like this

<details><summary>Screenshot 2</summary>

![image](https://user-images.githubusercontent.com/66517106/165736924-3f03e142-031b-4327-ae71-cad7ed64a9c1.png)

</details>
